src/System.ServiceProcess.ServiceController/src/Resources/Strings.resx

```diff
@@ -97,4 +97,7 @@
   <data name="Timeout" xml:space="preserve">
     <value>Time out has expired and the operation has not been completed.</value>
   </data>
+  <data name="PlatformNotSupported_ServiceController" xml:space="preserve">
+    <value>ServiceController allows to manipulate and access Windows services, it is not applicable for other operating systems.</value>
```

Comment:
Nits:
- "ServiceController allows to manipulate and access" => "ServiceController enables manipulating and accessing"
- ", it" => ". It" or ", and it"